terraso_backend/apps/graphql/schema/groups.py

```diff
@@ -82,6 +84,21 @@ class Meta:
         interfaces = (relay.Node,)
         connection_class = TerrasoConnection
 
+    def resolve_account_membership(self, info):
+        user = info.context.user
+        if user.is_anonymous:
+            return None
+        if hasattr(self, "account_memberships"):
+            if len(self.account_memberships) > 0:
+                return self.account_memberships[0]
+            return None
+        return self.memberships.filter(user=user).first()
```

Comment:
issue: ensure only approved members are included

Reply:
@paulschreiber for this one getting the pending membership is expected